currency_cost.py

```diff
@@ -0,0 +1,46 @@
+"""
+Cost object for currency
+"""
+
+import collections
+
+class CurrencyCost:
+    """
+    Save information in a currency trading.
+    """
+    def __init__(self, region, instruments):
+        self.asset = collections.namedtuple('asset', ['value', 'quantity'])
+        self.cost = 0
+        self.cur_asset = {
+            instrument : self.asset(0, 0) for instrument in instruments}
+        self.cur_asset[region] = self.asset(1, 1)
```

Comment:
I think PnL of our first day should be 0 anyway. No need to count our initial trade into PnL.